scripts/rosdistro_build_cache

```diff
@@ -94,11 +92,7 @@ def main():
         # write the cache
         data = yaml.dump(cache.get_data(), Dumper=CacheYamlDumper)
 
-        # On Python 3, we must encode the unicode yaml str prior to writing it,
-        # whereas for Python 2, the yaml output is a str which cannot be further
-        # encoded (compared with a unicode object).
-        if sys.version_info[0] >= 3:
-            data = data.encode('utf-8')
+        data = data.encode('utf-8')
```

Comment:
data = yaml.dump(cache.get_data(), Dumper=CacheYamlDumper).encode()

Reply:
Would omitting `'utf-8'` constitute a change in behavior on any platforms?

Reply:
It's the default: https://docs.python.org/3/library/stdtypes.html#str.encode

Reply:
A better way still might be to just let the encode happen implicitly by changing `open('wb')` to just `open('w')`, as it looks like gzip.open also has a textual mode that presumable does the same thing: https://docs.python.org/3/library/gzip.html#gzip.open

But I can also understanding wanting to keep this change simple. :)

Reply:
I'll take the suggestion, but let's not rock the boat too much :) a1b4ba9